docs/contributor-guide/development.md

```diff
@@ -0,0 +1,34 @@
+<!-- Source: https://github.com/arduino/tooling-project-assets/blob/main/documentation-templates/contributor-guide/other/development.md -->
+
+# Development Guide
+
+## Prerequisites
+
+The following development tools must be available in your local environment:
+
+- [Go](https://go.dev/dl/)
+- [Taskfile](https://taskfile.dev/)
+- [ddb client]//https://developer.android.com/tools/adb) [optionally]
+
+## Building the Project
+
+- `task init`
+
+## Uploading the arduino-app-cli into the board
+Connect an [Arduino UNO Q](https://www.arduino.cc/product-uno-q) board to the PC via USB.
```

Comment:
```suggestion
Connect an [Arduino UNO Q](https://docs.arduino.cc/hardware/uno-q/) board to the PC via USB.
```

From experience with the way the Arduino website is maintained, the previous URL is likely to stop working at some point in the future. The documentation pages are the most stable link targets (they typically only break when the URL changes after the product is retired).